Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -542,6 +552,19 @@ int Sequencer::run() {
               context.fpeMonitor = nullptr;
             }
 
+            if (enableEventTiming) {
```

Comment:
```suggestion
            if (!m_cfg.outputEventTimingFile.empty()) {
```

Reply:
Is it better to use `.empty()` everywhere rather than using a boolean? 
I would naïvely expect that using `.empty()` would have more negative impact on the performance than using a boolean.

Reply:
I think `empty` would just check against the size which results in an integer zero check so there should not be any overhead. Also, I believe the path is not that hot to leverage these kind of optimization.

Reply:
I personally prefer setting a `boolean` and using it everywhere as I am less likely to write a wrong condition by mistake by using it. But that might not be the best way to do it.
Should I change it to `.empty()` instead or leave it like that?

Reply:
No need to change it - also just a personal preference on my side